src/api_server.cc

```diff
@@ -124,10 +124,14 @@ void MetadataApiServer::HandleMonitoredResource(
       LOG(INFO) << "Found resource for " << id << ": " << resource;
     }
     conn->set_status(HttpServer::connection::ok);
+
+    std::string response = resource.ToJSON()->ToString();
+
     conn->set_headers(std::map<std::string, std::string>({
       {"Content-Type", "application/json"},
+      {"Content-Length", std::to_string(response.length())},
```

Comment:
I believe we use `std::string::size` everywhere else...

Reply:
Done.